packages/storebuilder/src/wizards/WizardHeader.tsx

```diff
@@ -1,7 +1,8 @@
 import { WizardHeader as WmeWizardHeader, Logo, ExitButton } from '@moderntribe/wme-ui';
 import { __ } from '@wordpress/i18n';
 import { useWizard } from '@store/hooks';
-import { StoreBuilderLogo } from '@store/logos';
+import { NexcessLogo } from '@store/logos';
```

Comment:
I may be incorrect (and I know the designs show it) but I don't think it makes sense to use the Nexcess logo in the wizard. Please check with Josh, Nell and/or Donaldo to be sure.

Reply:
Thanks @chumpgrub I will check that in.